packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -1,188 +1,193 @@
-        </PropertyGroup>
-
-        <CreateNewImage ContainerizeDirectory="$(ContainerizeFolder)"
-                        ToolPath="$(DotNetHostDirectory)"
-                        ToolExe="$(DotNetHostFileName)"
-                        BaseRegistry="$(ContainerBaseRegistry)"
-                        BaseImageName="$(ContainerBaseName)"
-                        BaseImageTag="$(ContainerBaseTag)"
-                        LocalContainerDaemon="$(LocalContainerDaemon)"
-                        OutputRegistry="$(ContainerRegistry)"
-                        ImageName="$(ContainerImageName)"
-                        ImageTags="@(ContainerImageTags)"
-                        PublishDirectory="$(PublishDir)"
-                        WorkingDirectory="$(ContainerWorkingDirectory)"
-                        Entrypoint="@(ContainerEntrypoint)"
-                        EntrypointArgs="@(ContainerEntrypointArgs)"
-                        Labels="@(ContainerLabel)"
-                        ExposedPorts="@(ContainerPort)"
-                        ContainerEnvironmentVariables="@(ContainerEnvironmentVariables)"
-                        ContainerRuntimeIdentifier="$(ContainerRuntimeIdentifier)"
-                        RuntimeIdentifierGraphPath="$(RuntimeIdentifierGraphPath)"> <!-- The RID graph path is provided as a property by the SDK. -->
-            <Output TaskParameter="GeneratedContainerManifest" PropertyName="GeneratedContainerManifest" />
-            <Output TaskParameter="GeneratedContainerConfiguration" PropertyName="GeneratedContainerConfiguration" />
-        </CreateNewImage>
-    </Target>
-</Project>
\ No newline at end of file
+    <CreateNewImage ContainerizeDirectory="$(ContainerizeFolder)"
+                    ToolPath="$(DotNetHostDirectory)"
+                    ToolExe="$(DotNetHostFileName)"
+                    BaseRegistry="$(ContainerBaseRegistry)"
+                    BaseImageName="$(ContainerBaseName)"
+                    BaseImageTag="$(ContainerBaseTag)"
+                    LocalContainerDaemon="$(LocalContainerDaemon)"
+                    OutputRegistry="$(ContainerRegistry)"
+                    ImageName="$(ContainerImageName)"
+                    ImageTags="@(ContainerImageTags)"
+                    PublishDirectory="$(PublishDir)"
+                    WorkingDirectory="$(ContainerWorkingDirectory)"
+                    Entrypoint="@(ContainerEntrypoint)"
+                    EntrypointArgs="@(ContainerEntrypointArgs)"
+                    Labels="@(ContainerLabel)"
+                    ExposedPorts="@(ContainerPort)"
+                    ContainerEnvironmentVariables="@(ContainerEnvironmentVariables)"
+                    ContainerRuntimeIdentifier="$(ContainerRuntimeIdentifier)"
+                    RuntimeIdentifierGraphPath="$(RuntimeIdentifierGraphPath)">
+      <!-- The RID graph path is provided as a property by the SDK. -->
```

Comment:
This comment was supposed to be tied to the usage of the property, but it can't be 'inline' - I wonder where we could put it otherwise.

Reply:
I put it as it was before for time being - thanks for catching!
Like the safest place is before `CreateNewImage` element.